source/baikal/index.md

```diff
@@ -18,12 +18,13 @@ Protect your privacy by hosting calendars and contacts yourself - with Baïkal.
 
 * [Download][1]
 * [Installation instructions][2]
+* [Installation using Docker (unofficial)][10]
 * [Upgrade instructions][3]
 
 <img src='/img/baikal-in-use.png' style="width: 100%; max-width: 640px;" />
 
 Baïkal is free and [open-source][4], developed by [Net Gusto][5] and [fruux][6]
-together with many [contributors][7]. It is released under the GNU GPL v3 license.  
+together with many [contributors][7]. It is released under the GNU GPL v3 license.
```

Comment:
The space characters here are there on purpose. They create a line break without creating a new paragraph.

Reply:
Fixed.